src/RequestHandler/RequestHandler.vcxproj

```diff
@@ -231,13 +231,31 @@
     <ClCompile Include="outofprocess\websockethandler.cxx" />
     <ClCompile Include="outofprocess\winhttphelper.cxx" />
   </ItemGroup>
+  <Target Name="CreateVersionHeader" BeforeTargets="PrepareForBuild">
```

Comment:
Does this break incremental compilation? I'm not sure if this is a thing in vcsproj.

Reply:
This is what used to be in the ANCM repo: https://github.com/aspnet/AspNetCoreModule/blob/e714153782a9fde9a529804efb0bcbf015af20b1/src/AspNetCore/AspNetCore.vcxproj#L193. So it is probably available.